src/box/lua/console.lua

```diff
@@ -299,7 +312,10 @@ local param_handlers = {
     o = set_output,
     delimiter = set_delimiter,
     delim = set_delimiter,
-    d = set_delimiter
+    d = set_delimiter,
+    continuation = set_continuation,
+    con = set_continuation,
```

Comment:
I couldn't decide what is more clear `con` or `cont`, so I left it as is. If @Totktonada has a strong opinion I can change it either way.

Reply:
Just `\set continuation` is okay.

Reply:
Now `\set continuation on|off` is implemented as well as the `set c on|off alias. I would leave only former, because `c` may point many different things (connection, console, continue, column, ...) and we may want to use it for something more frequent in a future.

Reply:
Removed the alias, as we discussed.

Reply:
AFAIS, the original and the following questions are resolved, so I'm resolving the discussion.